libs/ledger/src/protocols/main_chain_rpc_service.cpp

```diff
@@ -463,6 +436,7 @@ State MainChainRpcService::OnWaitForBlocks()
 
     state_machine_->Delay(std::chrono::milliseconds{100 * consecutive_failures_});
     return State::REQUEST_NEXT_BLOCKS;
+  default:;
```

Comment:
I would remove default case, otherwise you won’t get compiler warning when the enum is changed